cachelib/cachebench/cache/CacheStats.h

```diff
@@ -116,6 +118,12 @@ struct Stats {
         << std::endl;
     out << folly::sformat("RAM Evictions : {:,}", numEvictions) << std::endl;
 
+    for (auto pid = 0U; pid < poolUsageFraction.size(); pid++) {
```

Comment:
This is only tracking per-pool usage stats right? Can you update the PR title to reflect this?

Reply:
done